tutorials/e2e-huggingface-summarization/e2e-dl-huggingface-summarization.ipynb

```diff
@@ -0,0 +1,1103 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "1dbce700",
+   "metadata": {},
+   "source": [
+    "# Using Azure ML Pipelines to Train and Use HuggingFace models for Text Summarization Tasks"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0bbe4d4d",
+   "metadata": {},
+   "source": [
+    "**Learning Objectives** \n",
+    "By the end of this two part tutorial, you should be able to use Azure Machine Learning (AML) to finetune Hugging Face NLP models.\n",
+    "\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4c00c3c3",
+   "metadata": {},
+   "source": [
+    "**Requirements**\n",
+    "In order to benefit from this tutorial, you need to have:\n",
+    "- basic understanding of Machine Learning projects workflow\n",
+    "- an Azure subscription. If you don't have an Azure subscription, [create a free account](https://aka.ms/AMLFree) before you begin.\n",
+    "- a working AML workspace. A workspace can be created via Azure Portal, Azure CLI, or Python SDK. [Read more](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-manage-workspace?tabs=python).\n",
+    "- a Python environmnet\n",
+    "- [installed Azure Machine Learning Python SDK v2](https://github.com/Azure/azureml-examples/blob/sdk-preview/sdk/setup.sh)\n",
+    "- familiarity with Hugging Face framework\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0113b218",
+   "metadata": {},
+   "source": [
+    "**Motivations** \n",
+    "In this tutorial, we will create an AML pipeline to finetune a huggingface model in AML. Specifically, we train a light bert model on CNN News dataset to perform text summarization . We evaluate the trained model's performance on a specific domain (here Medical), then finetune the model on the medical domain, and compare the results.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "550fe006",
+   "metadata": {},
+   "source": [
+    "### Connect to AzureML\n",
+    "\n",
+    "Before we dive in the code, we'll need to create an instance of MLClient to connect to Azure ML. Please provide the references to your workspace below.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b2e510cb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Handle to the workspace\n",
+    "from azure.ai.ml import MLClient\n",
+    "\n",
+    "# Authentication package\n",
+    "from azure.identity import DefaultAzureCredential, InteractiveBrowserCredential\n",
+    "\n",
+    "try:\n",
```

Comment:
Unsure: didn't we rule out using try/except as part of the feedback from Akshaya on the azureml 101 notebook?

Reply:
the pipeline samples are all using this. Let's update all together once a new format is confirmed. We can ask Bala to verify/update all.